### PR TITLE
Remove createdDate from mutation

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: ts/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nationalarchives/tdr-generated-graphql",
-  "version": "1.0.28",
+  "version": "1.0.29",
```
Comment (TomJKing) [inline ts/package.json:3]: Isn't this handled by the Jenkins build. Generates a new PR with version updated

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.64-SNAPSHOT"
+version in ThisBuild := "0.0.65-SNAPSHOT"
```
Comment (TomJKing) [inline version.sbt:1]: Isn't this handled by the Jenkins build. Generates a new PR with version updated

**Review by TomJKing — COMMENTED** [review]
Couple of minor comments

File: ts/package.json (hunk 1 above)
Comment (MancunianSam) [reply ts/package.json:3]: Same as the other ones, I needed them deployed to get the others to run.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**